content/trusted-content/official-images/using.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Using Docker Official Images
+description: |
+  Learn about building applications with Docker Official images
+  and how to interpret the tag names they use.
+keywords: docker official images, doi, tags, slim, feedback, troubleshooting
+---
+
+If you are new to Docker, it's recommended you use the Docker Official Images
+in your projects. These images have clear documentation, promote best
+practices, and are designed for the most common use cases. Advanced users can
+review Docker Official Images as part of your `Dockerfile` learning process.
+
+A common rationale for diverging from Docker Official Images is to optimize for
+image size. For instance, many of the programming language stack images contain
+a complete build toolchain to support installation of modules that depend on
+optimized code. An advanced user could build a custom image with just the
+necessary pre-compiled libraries to save space.
+
+## Tags
+
+The repository description for each Docker Official Image contains a
+**Supported tags and respective Dockerfile links** section that lists all the
+current tags with links to the Dockerfiles that created the image with those
+tags. The purpose of this section is to show what image variants are available.
+
+![Supported tags for Ubuntu](../images/supported_tags.webp)
+
+Tags listed on the same line all refer to the same underlying image. Multiple
+tags can point to the same image. For example, in the `ubuntu` Docker Official
+Images repository, the tags `24.04`, `noble-20240225`, `noble`, and `devel` all
+refer to the same image.
+
+The `latest` tag for a Docker Official Image is often optimized for ease of use
+and includes a wide variety of useful software, such as Git and build tools.
+Because of their ease of use and wide applicability, `latest` images are often
+used in getting-started guides, but they're not recommended for production use.
+
+## Slim images
```

Comment:
Yes that would be good. We also have this page which is related, but a little bit of an oddity in the "best practices" section: https://docs.docker.com/build/building/base-images/

Not sure what to do about that if anything. Just an observation... Maybe there's a home for "base images" somewhere.

Reply:
Oh, interesting; I recall it used to be debian (as a default?), but maybe that changed (debian generally considered more "stable", but ubuntu having more up to date packages)

Reply:
If we look at repositories instead of individual tags, we get a slightly different story (the point about Ubuntu still holds, but it also helps highlight that Eclipse Temurin is a special case because Java is, not because Temurin itself really is):

```console
$ bashbrew children --uniq debian | cut -d: -f1 | sort -u | wc -l
66
$ bashbrew children --uniq ubuntu | cut -d: -f1 | sort -u | wc -l
45
$ bashbrew children --uniq eclipse-temurin | cut -d: -f1 | sort -u | wc -l
23
```